lib/javascripts/base_app.js

```diff
@@ -144,6 +144,10 @@ BaseApp.prototype = {
     }
 
     this._storage.set(keyOrObject, value);
+  },
+
+  modal: function(url) {
```

Comment:
Maybe there's a way we can provide an actual shim, by extending jQuery and taking a template name as an argument? Then the Base constructor could handle switching to the template if the location is modal? We could attempt to pass the data serialised as a url  parameter? This code is fine as an example implementation but I'm not sure it belongs here.